src/js/shield.js

```diff
@@ -382,8 +399,15 @@ export function generateShieldCtx(map, id) {
   // Convert numbering systems. Normally alternative numbering systems should be
   // tagged directly in ref=*, but some shields use different numbering systems
   // for aesthetic reasons only.
-  if (routeDef.ref && shieldDef.numberingSystem === "roman") {
-    routeDef.ref = romanizeRef(routeDef.ref);
+  if (routeDef.ref) {
+    switch (shieldDef.numberingSystem) {
+      case "arab":
+        routeDef.ref = latinToArabicDigits(routeDef.ref);
+        break;
```

Comment:
`"arab"` was chosen here to match [Unicode CLDR](https://github.com/unicode-org/cldr/blob/main/common/bcp47/number.xml#L15).

Reply:
Should it be `arabicizeRef()`? :grin:

Reply:
Either that or `arabizeRef()`, yes.